pinot-spi/src/main/java/org/apache/pinot/spi/config/table/SegmentsValidationAndRetentionConfig.java

```diff
@@ -29,6 +29,7 @@
 public class SegmentsValidationAndRetentionConfig extends BaseJsonConfig {
   private String _retentionTimeUnit;
   private String _retentionTimeValue;
+  private String _deletedSegmentRetentionPeriod;
```

Comment:
Also change the getter and setter
```suggestion
  private String _deletedSegmentsRetentionPeriod;
```